docker/Dockerfile.contrib

```diff
@@ -0,0 +1,31 @@
+FROM node AS build
+
+COPY node-zwave-js /root/node-zwave-js
+COPY zwavejs2mqtt /root/zwavejs2mqtt
+
+WORKDIR /root/node-zwave-js
+RUN npm install
```

Comment:
```suggestion
RUN npm install -g npm@7
RUN npm install
```

Reply:
```
Step 11/18 : RUN npm install -g npm@7
 ---> Running in 7616e7296876
npm ERR! code EISDIR
npm ERR! syscall copyfile
npm ERR! path /usr/local/lib/node_modules/npm
npm ERR! dest /usr/local/lib/node_modules/.npm-i9nnxROI
npm ERR! errno -21
npm ERR! EISDIR: illegal operation on a directory, copyfile '/usr/local/lib/node_modules/npm' -> '/usr/local/lib/node_modules/.npm-i9nnxROI'

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-11-26T15_51_43_673Z-debug.log
The command '/bin/sh -c npm install -g npm@7' returned a non-zero code: 235
```

Reply:
Hummm is there a node image with npm 7 already installed?